pkg/version/ttn.go

```diff
@@ -3,4 +3,4 @@
 package version
 
 // TTN Version
-var TTN = "v3.3.1-dev"
+var TTN = "3.3.1-dev"
```

Comment:
Not related to this PR, but worth a quickfix as well I guess - shouldn't we rename this variable? E.g. `Stack` or `TTS`?

Reply:
Not in this PR, but yes. I created an issue for you: https://github.com/TheThingsNetwork/lorawan-stack/issues/1656